model/src/ww3_strt.F90

```diff
@@ -816,10 +816,19 @@ PROGRAM W3STRT
     !--- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
     ! 8.  ITYPE = 5, fetch limited JONSWAP.
     !
-  ELSE
+  ELSE IF ( ITYPE .EQ. 5 ) THEN
     INXOUT = 'CALM'
     IF ( IAPROC .EQ. NAPOUT ) WRITE (NDSO,980)
     !
+  ELSE IF ( ITYPE .EQ. 6 ) THEN
+    INXOUT = 'FTCH'
+    
+    CALL NEXTLN ( COMSTR , NDSI , NDSEN )
+    READ (NDSI,*,END=801,ERR=802) FETCH
```

Comment:
After the recent PR we need to update this to be: 

READ (NDSI,*,IOSTAT=IERR) FETCH 
IF (IERR.NE.0) CALL EXTIOF(NDSE,IERR,'W3STRT','INPUT',11)